### DISCUSSION TITLE
How does RDKit fingerprint compares to FMorgan fingerprints

### DISCUSSION THREAD

**Discussion by kathputu** [rdkit/rdkit#6156] [General]
Hi all,

I am trying to do some analysis on our chemical library. I've tried using different fingerprints to capture molecule information but different fingerprints give different results/information. 

I am wondering how does the RDKit fingerprint differ from the feature morgan (FCFP) fingerprints? Was anyone able to find any documentation on what the RDKit fingerprints encode and what the FCFP fingerprints encode?

Thank you!

**Comment by greglandrum** [comment]
This is in the RDKit documentation: https://www.rdkit.org/docs/RDKit_Book.html#additional-information-about-the-fingerprints
